spec/unit/actions/app_start_spec.rb

```diff
@@ -89,17 +89,24 @@ module VCAP::CloudController
               state:   DropletModel::STAGED_STATE,
             )
           end
-          let(:package) { PackageModel.make(app: app, package_hash: 'some-awesome-thing', state: PackageModel::READY_STATE) }
+          let(:package) do
+            PackageModel.make(
+              app: app,
+              package_hash: 'some-sha1',
+              sha256_checksum: 'some-sha256',
+              state: PackageModel::READY_STATE
```

Comment:
One thought I had while working on this was what if we could key off of the `PackageModel::READY_STATE` instead of the presence of checksums for know when an app is ready to be staged in v2?

I'm unsure of if there are other implications around doing that in v2 though...
Or if `READY` truly means "ready" in this case

Reply:
I think principal-of-smallest change is probably best here, and sticking with using the checksums fits with that.